GliaWidgets/Sources/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -264,37 +264,6 @@ class ChatViewModel: EngagementViewModel, ViewModel {
             break
         }
     }
-
-    override func stateChanged(_ state: InteractorState) {
```

Comment:
Is this no longer needed because of changes in `EngagementViewModel`?

Reply:
Yeap, exactly